wechatpay/build.gradle

```diff
@@ -11,6 +11,7 @@ plugins {
     alias libs.plugins.kotlin.android
     alias libs.plugins.kotlin.parcelize
     alias libs.plugins.dokka.convention
+    alias libs.plugins.generate.dependency.list
```

Comment:
Comment for later: it would be nice to create a shared plugin in buildSrc which automatically imports all common plugins for our releasable libraries. This would help us avoid having to add an import to every single `build.gradle` and risking forgetting to add it in some places.